javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,38 @@
+var width = 23;
+var height = 17;
+var area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+var payment = 150;
+var bill = 100;
+var change = payment - bill;
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+var quizzes = 96;
+var midTerm = 92;
+var final =  90;
+var grade = (quizzes + midTerm + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+var firstName = 'Alvaro'
+var lastName = 'Diaz'
+var fullName = firstName + ' ' + lastName;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+var pH = 9;
+var isAcidic = pH < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+var headCount = 15;
+var isSparta = headCount === 300;
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+var motto = fullName += ' ' + 'is the GOAT' ;
```

Comment:
You should be using the addition assignment operator after creating the `motto` variable for the first time. With the way you've written this, the value of the `fullName` variable will change too, which is not the intention.